work/worker_fake.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 The klaytn Authors
+// This file is part of the klaytn library.
+//
+// The klaytn library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The klaytn library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.]
+
+package work
+
+import (
+	"github.com/klaytn/klaytn/blockchain/state"
+	"github.com/klaytn/klaytn/blockchain/types"
+)
+
+type FakeWorker struct{}
+
+// NewFakeWorker disables mining and block processing
+//
```

Comment:
Is this line intended?

Reply:
Yes. Do you want it to be erased?

Reply:
Yes, I rarely see the comments with an empty blank line between the lines in our code.
It would be better to be removed for consistency.

Reply:
review reflected